src/current/_includes/releases/v23.1/v23.1.14.md

```diff
@@ -0,0 +1,100 @@
+
+<h3 id="v23-1-14-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where an active replication report update could get stuck in a retry loop on clusters with over 10000 ranges, which would prevent a node from shutting down cleanly. [#114242][#114242]
+- Fixed error handling for `GetFiles` so that it does not cause a nil pointer dereference. [#114829][#114829]
+- Fixed a bug that prevented the **SQL Activity** page from showing internal statements when the `sql.stats.response.show_internal.enabled` [cluster setting](https://www.cockroachlabs.com/docs/v23.1/cluster-settings) was set to `true`. [#114819][#114819]
+- Rolled back deletes no longer cause a discrepancy between computed stats and the actual stored values. [#113772][#113772]
+- Previously, CockroachDB could encounter an internal error when using the text format (as opposed to binary) when preparing statements with user-defined composite types. The bug was introduced in 23.1 and is now fixed. [#115063][#115063]
+- Fixed a bug that could cause cluster version finalization to contend with descriptor lease renewals on large clusters. Descriptor lease renewals previously had a higher priority than cluster upgrade finalization. Finalization now always has a higher priority than descriptor lease renewal. [#114672][#114672]
+- Fixed a bug that would cause a [prepared statement](https://www.cockroachlabs.com/docs/v23.1/sql-grammar#prepare_stmt) to fail if it references an enum as well as a table that has undergone a schema change. [#115174][#115174]
+- [Backups](https://www.cockroachlabs.com/docs/v23.1/backup) now evenly distribute their work across all replicas, including followers, regardless of leaseholder placement. [#115018][#115018]
+- Replica processing in store queues has been enhanced to smoothly handle replicas that have undergone ID changes or destruction. These replicas can now be effectively re-added and processed without encountering previous purgatory-related issues. [#115036][#115036]
+- Fixed an issue that could lead to stuck queries or inaccurate results when using lookup [joins](https://www.cockroachlabs.com/docs/v23.1/joins) involving equality columns and multiple ranges. The bug was introduced in 22.2 and is now fixed. [#115581][#115581]
+- [`CREATE INDEX`](https://www.cockroachlabs.com/docs/v23.1/create-index) with expressions no longer fail on materialized [views](https://www.cockroachlabs.com/docs/v23.1/views) when the declarative schema changer is used. [#115596][#115596]
+- Fixed a bug that could result in an incorrect "too few columns" error for queries that use `ANY <array>` syntax with a subquery. [#115591][#115591]
+- Fixed a bug that could cause "too few/many columns" errors for queries that used `IN` or `NOT IN` with a non-trivial right operand, such as a subquery (rather than a constant tuple). [#115591][#115591]
```

Comment:
```suggestion
- Fixed a bug that could cause  a "too few/many columns" errors for a query that used an `IN` or `NOT IN` clause with a non-trivial right operand, such as a subquery (rather than a constant tuple). [#115591][#115591]
```